test/e2e/config/nutanix.yaml

```diff
@@ -153,26 +153,24 @@ variables:
   NUTANIX_PORT: "9440"
   NUTANIX_INSECURE: true
   NUTANIX_ADDITIONAL_TRUST_BUNDLE: ""
-  KUBERNETES_VERSION: "v1.30.2"
-  KUBERNETES_VERSION_v1_27: "v1.27.13"
-  KUBERNETES_VERSION_v1_28: "v1.28.9"
-  KUBERNETES_VERSION_v1_29: "v1.29.6"
-  KUBERNETES_VERSION_v1_30: "v1.30.2"
+  KUBERNETES_VERSION: "v1.31.0"
+  KUBERNETES_VERSION_v1_28: "v1.28.13"
+  KUBERNETES_VERSION_v1_29: "v1.29.8"
+  KUBERNETES_VERSION_v1_30: "v1.30.4"
   NUTANIX_SSH_AUTHORIZED_KEY: ""
   CONTROL_PLANE_ENDPOINT_IP: ""
   CONTROL_PLANE_ENDPOINT_IP_V124: ""
   CONTROL_PLANE_MACHINE_COUNT: 1
   WORKER_MACHINE_COUNT: 1
   NUTANIX_PRISM_ELEMENT_CLUSTER_NAME: ""
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME: "ubuntu-2204-kube-v1.30.2.qcow2"
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_27: "ubuntu-2204-kube-v1.27.13.qcow2"
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_28: "ubuntu-2204-kube-v1.28.9.qcow2"
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_29: "ubuntu-2204-kube-v1.29.6.qcow2"
-  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_30: "ubuntu-2204-kube-v1.30.2.qcow2"
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME: "rockylinux-9-kube-v1.31.0.qcow2"
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_28: "rockylinux-9-kube-v1.28.13.qcow2"
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_29: "rockylinux-9-kube-v1.29.8.qcow2"
+  NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_30: "rockylinux-9-kube-v1.30.4.qcow2"
```

Comment:
we should add NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME_v1_31 as well or planning to add when we add test?

Reply:
Good point. It's basically same as the default i.e. `NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME`. So, we can add it later.